docs/troubleshooting/semgrep.md

```diff
@@ -50,13 +50,22 @@ semgrep:
 
 We currently don't have instructions for other providers, but logs for the action are always saved in `.semgrep_logs/`. There will be two files, `semgrep_agent_logs` and `semgrep_agent_output`. The former is a more verbose logging of what happened; the second contains the output that the action collected for semgrep. If you are running in docker, you can find the logs there.
 
+## Troubleshooting Semgrep CI
 
-## Troubleshooting Semgrep CLI 
+### Reproducing the run locally
+
+It is possible to reproduce some parts of Semgrep CI locally to aid in debugging through the following steps:
+
+- First go to the [API token page](https://semgrep.dev/orgs/semgrep/settings/tokens) and create a new API token
+- Run `semgrep login` on your machine which will ask you to paste your API key
+- Run `SEMGREP_REPO_NAME=<your-org-here>/<repo-name-here> semgrep --config policy` (i.e. `SEMGREP_REPO_NAME=returntocorp/semgrep semgrep --config policy` for the github repo `returntocorp/semgrep`). This will fetch the rules configured on any semgrep.dev policy for this repo and run it locally
```

Comment:
```suggestion
- Run `SEMGREP_REPOSITORY_NAME=<your-org-here>/<repo-name-here> semgrep --config policy` (For example, `SEMGREP_REPOSITORY_NAME=returntocorp/semgrep semgrep --config policy` would be used for the GitHub repository `returntocorp/semgrep`). This will fetch the rules configured on any Semgrep App policy for this repository and run a local Semgrep scan using those rules.
```